server/service/src/sync/translations/purchase_order.rs

```diff
@@ -458,8 +463,8 @@ impl SyncTranslation for PurchaseOrderTranslation {
             communications_charge,
             insurance_charge,
             freight_charge,
-            supplier_discount_amount,
-            purchase_order_number,
+            supplier_discount_amount: line_total_after_discount
+                * (supplier_discount_percentage.unwrap_or(0.0) / 100.0),
```

Comment:
This makes sense - but I was wondering is there any risk of weird floating point errors resulting from different calculation methods between sql vs rust?